### PR TITLE
Functional requirement

### PR DESCRIPTION
**Pull request by flyfl**
Fixing #160.

### CODE REVIEW

**Comment by acBerger** [comment]
[PDF] (http://flyfl.github.io/Beagle/branches/FunctionalRequirement/Requirements%20Specification.pdf)

**Comment by viridispirus** [comment]
lgtm

File: doc/Requirements Specification/Parts/Functional Requirements.lyx [hunk 1]
```diff
@@ -838,14 +806,21 @@ gls{PCM}
...
+
+
+\backslash
+gls{PCM}
+\end_layout
 
 \end_inset
 
-.
+ will only be extended.
```
Comment (acBerger) [inline doc/Requirements Specification/Parts/Functional Requirements.lyx]: For me this sounds like we extend the modified (= result PCM). If I got this right, this is not what we really do.

**Comment by flyfl** [comment]
Look at #186. Excluded the changes in Discussion chapter

**Pull request closed without merging**